samples/cpp/image_generation/README.md

```diff
@@ -148,3 +149,63 @@ The resuling image is:
    ![](./inpainting.bmp)
 
 Note, that LoRA, heterogeneous execution and other features of `Text2ImagePipeline` are applicable for `InpaintingPipeline`.
+
+## benchmarking sample for text to image pipeline
+
+This `benchmark_image.cpp` sample script demonstrates how to benchmark the text to image pipeline, image to image pipeline and inpainting pipeline. The script includes functionality for warm-up iterations, generating image, and calculating various performance metrics.
+
+The usage of this sample is:
+```bash
+./benchmark_image [OPTIONS]
+```
+Options:
+- `-t, --pipeline_type` (default: `"text2image"`): Pipeline type(text2image, image2image, inpainting).
+- `-m, --model`: Path to the model and tokenizers base directory.
+- `-p, --prompt` (default: `"The Sky is blue because"`): The prompt to generate text.
+- `--nw, --num_warmup` (default: `1`): Number of warmup iterations.
+- `-n, --num_iter` (default: `3`): Number of iterations.
+- `-d, --device` (default: `"CPU"`): Device to run the model on.
+- `--wh, --width` (default: `512`): The width of the output image.
+- `--ht, --height` (default: `512`): The height of the output image.
```

Comment:
```suggestion
- `--h, --height` (default: `512`): The height of the output image.
```

Reply:
`-h` refers to `help`, so the abbreviation of height here needs to be `ht`